dftimewolf/lib/collectors/gcloud.py

```diff
@@ -314,4 +373,96 @@ def _FindDisksToCopy(self) -> List[compute.GoogleComputeDisk]:
 
     return disks_to_copy
 
-modules_manager.ModulesManager.RegisterModule(GoogleCloudCollector)
+class GKEDiskCopier(GoogleCloudCollector):
+
+  def SetUp(self,
+            analysis_project_name: str,
+            remote_project_name: str,
+            remote_cluster_name: str,
+            remote_cluster_zone: str,
+            workload_name: Optional[str]=None,
+            workload_namespace: Optional[str]=None,
+            incident_id: Optional[str]=None,
+            zone: str='us-central1-f',
+            create_analysis_vm: bool=True,
+            boot_disk_size: float=50,
+            boot_disk_type: str='pd-standard',
+            cpu_cores: int=4,
+            image_project: str='ubuntu-os-cloud',
+            image_family: str='ubuntu-1804-lts') -> None:
+    """Sets up a GKE disk collector.
+
+    This method creates and starts an analysis VM in the analysis project and
+    selects nodes whose boot disks will be copied from the remote cluster.
+
+    If both the workload_name and workload_namespace are specified, only the
+    nodes supporting the workload's pods will be copied. If they are not
+    specified, all the nodes' disks will be copied to the analysis VM.
+
+    If analysis_project_name is not specified, analysis_project will be same
+    as remote_project.
+
+    Args:
+      analysis_project_name (str): Optional. name of the project that contains
```

Comment:
here and everywhere else, since we have type annotations, adding the type to the doc string is obsolete, you should remove all these